### PR TITLE
document CO `[ChannelN],input_configured`

### PR DESCRIPTION
**Pull request by Swiftb0y**
As pointed out by @ronso0 in https://github.com/mixxxdj/mixxx/issues/13252#issuecomment-2127921807

### CODE REVIEW

**Comment by Swiftb0y** [comment]
wording might need a little work. Suggestions welcome

File: source/chapters/appendix/mixxx_controls.rst [hunk 1]
```diff
@@ -2876,10 +2876,14 @@ Then you can use your :term:`MIDI` controller to control its volume and some oth
...
 
-.. mixxx:control:: [MicrophoneN],input_configured
+.. mixxx:control:: [ChannelN],input_configured
+                   [MicrophoneN],input_configured
                    [AuxiliaryN],input_configured
 
    1 if there is input is configured for this channel, 0 if not.
+   In the case of :mixxx:cogroupref:`[ChannelN]` it corresponds to
+   Vinyl Control. This should be checked before enabling :mixxx:coref:`[ChannelN],passthrough`
+   as you can't passthrough an input channel that is not configured.
```
Comment (ronso0) [inline source/chapters/appendix/mixxx_controls.rst:2886]:
```suggestion
   Vinyl Control. A configured input is required to enable :mixxx:coref:`[ChannelN],passthrough`
```
?

**Review by ronso0 — COMMENTED** [review]
Thank you.

**Comment by ronso0** [comment]
Thank you, LGTM

**Comment by Swiftb0y** [comment]
Thank you

**Pull request merged**